recipe/meta.yaml

```diff
@@ -1,34 +1,33 @@
-{% set author = "slundberg" %}
 {% set name = "shap" %}
-{% set version = "0.42.1" %}
-{% set sha256 = "3cba8049c36b41c2319270ab62e1d54135b7a6d76705cffbda3c6067aacadb29" %}
+{% set version = "0.46.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/{{ author }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bdaa5b098be5a958348015e940f6fd264339b5db1e651f9898a3117be95b05a0
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vv
-  missing_dso_whitelist:  # [win]
-   - $RPATH/cudart64_101.dll  # [win]
-  skip: true # [py<37 or s390x]
+  # s390x: numba not available
+  skip: true  # [py<39 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make  # [unix]
```

Comment:
I don't think make is needed. It's using pure setuptools stuff: https://github.com/shap/shap/blob/v0.46.0/setup.py#L101-L111

Reply:
thanks, that is right. Removed.